### DISCUSSION TITLE
Academy: "Administration of Pimcore" not accepting correct answers

### DISCUSSION THREAD

**Discussion by Myrmidonut** [pimcore/pimcore#9092] [Q&A / Support] [answered]
I have a strong suspicion that this question is not accepting correct answers anymore since the changes in the acedemy for the Pimcore X release (I started on the certificate on may 10th 2021 and I noticed one day later when Pimcore X was released that some questions/answers changed):

Pimcore Developer Junior Certificate - Administration of Pimcore - Question 6: How to deactivate maintenace mode?

These are my answers that the system doesn't accept which are all mentioned in the video (please remove the image if it is against the rules to show answers here):

![Question 6](https://user-images.githubusercontent.com/26467570/117834082-2a921080-b277-11eb-9422-be13f68da0fd.png)

**Comment by fashxp** [comment, marked as answer]
ups. thx for reporting. fixed now.
